Cargo.toml

```diff
@@ -26,7 +26,8 @@ deno_core = { path = "./vendor/deno/core", optional = true }
 deno_runtime = { path = "./vendor/deno/runtime", optional = true }
 deno = { path = "./vendor/deno/cli", optional = true }
 # clarity_repl = { package = "clarity-repl", path = "../../clarity-repl", features = ["cli"] }
-clarity_repl = { package = "clarity-repl", version = "=0.22.1" }
+# clarity_repl = { package = "clarity-repl", version = "=0.22.0" }
+clarity-repl = { git = "https://github.com/hirosystems/clarity-repl", branch = "develop" }
```

Comment:
Waiting on https://github.com/hirosystems/clarity-repl/pull/99 to be released

Reply:
Realized my other comment should have been here... You can update this to 0.23.0, which was just released. Thanks!